mailbox/tools/indexer/src/test/java/org/apache/mailbox/tools/indexer/ErrorRecoveryIndexationTaskSerializationTest.java

```diff
@@ -43,19 +43,20 @@
     private final int successfullyReprocessedMailCount = 42;
```

Comment:
We need test to ensure we can still deserialize the existing JSON fields without "mailbox failures"

Reply:
aren't the legacy tests supposed to do just that?